uiowa/uiowa
# [IWP] Migrate writer bio

Resolves https://github.com/uiowa/uiowa/issues/7699

# How to test

```
ddev blt ds --site=iwp.uiowa.edu && ddev drush @iwp.local en iwp_migrate -y && ddev drush @iwp.local uli admin/config/sitenow/sitenow-migrate
```

- Configure SiteNow Migrate following instructions for DevDesktop. 
  - Database: `iwp`.
  - User: `root`
  - Host: `host.docker.internal`
  - Port: `33067`
  - Base URL: `https://iwp.uiowa.edu`
- Check the migration status: `ddev drush @iwp.local ms`
- Run all migrations `ddev drush @iwp.local mim --all  `  
- or
- Import the countries: `ddev drush @iwp.local mim iwp_countries`
- Import the writer bios: `ddev drush @iwp.local mim iwp_bio`
- Import the redirects: `ddev drush @iwp.local mim iwp_bio_redirects`
- Rollback `ddev drush @iwp.local mr --all`

Go to https://iwp.uiowa.ddev.site/residency/participants to view all. 
Check against live site to make sure that redirects, alt text, files, etc have all been correctly migrated.
